plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoSelectAllWithCaseInsensitiveUpperCase.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.mongodb;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.mongodb.client.MongoClient;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.QueryRunner;
+import org.bson.Document;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.mongodb.MongoQueryRunner.createMongoClient;
+import static io.trino.plugin.mongodb.MongoQueryRunner.createMongoQueryRunner;
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static java.util.Locale.ENGLISH;
+
+public class TestMongoSelectAllWithCaseInsensitiveUpperCase
+        extends AbstractTestQueryFramework
+{
+    private MongoServer server;
+    private MongoClient client;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        server = new MongoServer();
+        client = createMongoClient(server);
+        return createMongoQueryRunner(server, ImmutableMap.of(), ImmutableMap.of(), ImmutableMap.of("mongodb.case-insensitive-name-matching", "false"), ImmutableList.of(), (s) -> {});
```

Comment:
Rename `(s)` to `setup`. Also, this line looks too long. Could you wrap per argument?